games-strategy/lgeneral/lgeneral-1.3.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A turn-based strategy engine"
+DESCRIPTION="LGeneral is a turn-based strategy engine heavily inspired by Panzer General. \
+You play single scenarios or whole campaigns turn by turn against a human \
+player or the AI. Entrenchment, rugged defense, defensive fire, surprise \
+contacts, surrender, unit supply, weather influence, reinforcements and other \
+implementations contribute to the tactical and strategic depth of the game."
+HOMEPAGE="http://lgames.sourceforge.net/"
+COPYRIGHT="2000, 2015 Michael Speck, Leo Savernik, Peter Ivanyi"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/lgeneral/lgeneral/lgeneral-$portVersion.tar.gz"
```

Comment:
Yes, this is "secure". We consider secure anything that gives us a file, so we can compute an SHA256 checksum.
We consider unsafe anything else, including git and svn checkouts, as they give a large set of file and there is no convenient way to compute a checksum out of that. So, in theory, people could replace the svn/git repo and we would have no way of knowing.